.gitignore

```diff
@@ -1,4 +1,8 @@
-s3watcher/node_modules
+S3Watcher/node_modules
```

Comment:
I suggest just having `node_modules` which is ignored recursively unless prefixed with a `/`.